x-pack/plugins/apm/server/routes/errors/get_error_groups/get_error_group_samples.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { rangeQuery, kqlQuery } from '@kbn/observability-plugin/server';
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import { asMutableArray } from '../../../../common/utils/as_mutable_array';
+import {
+  ERROR_GROUP_ID,
+  ERROR_ID,
+  SERVICE_NAME,
+  TRANSACTION_SAMPLED,
+} from '../../../../common/es_fields/apm';
+import { environmentQuery } from '../../../../common/utils/environment_query';
+import { APMEventClient } from '../../../lib/helpers/create_es_client/create_apm_event_client';
+
+const ERROR_SAMPLES_SIZE = 10000;
+
+export async function getErrorGroupSummary({
```

Comment:
Should this be called `getErrorGroupSamples`?

Reply:
Copy pasta mistake, fixed!